Aufgabe_2/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="de">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="widht=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./css/main.css">
+        <title>Ist noch offen!</title>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <ul class="navigation">
+                    <li><a href="#home">home</a></li>
+                    <li><a href="#about">about us</a></li>
+                    <li><a href="#team">our team</a></li>
+                    <li><a href="#contact">contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <article id="home" class="home">
+                <span class="headline"><H1> We bulid your <em>Infographics</em> in <br> les then <strong> ONE HOUR!! </strong></H1></span>
```

Comment:
h1 kannst du klein schreiben
span in den h1 tag rein
span ist ein inline element und für inline elemente zum stylen 
div ist für block elemente zum stylen